setup.py

```diff
@@ -0,0 +1,12 @@
+"""
+Run `conda env create --file environment.yaml` to create an editable 
+install of this project
+"""
+
+from setuptools import setup, find_packages
+
+setup(
+    name="Example Snowpark Python project",
+    version="0.1.0",
+    packages=find_packages()
+)
```

Comment:
Can a pyproject.toml with setuptools specified replace this setup.py and requirements.txt? I went with that in my project since pyproject.toml is the recommended way since PEP518 over setup.py. So the dependencies array would contain what's currently in requirements.txt. 


Possible pyproject.toml file to replace setup.py and requirements.txt:
```
[build-system]
requires = ["setuptools>=64", "wheel"]
build-backend = "setuptools.build_meta"

[project]
name = "Example Snowpark Python project"
version = "0.1.0"
dependencies = [
  "snowflake-snowpark-python",
  "tomli",
  "toml",
  "pytest"
  "snowflake-vcrpy @ git+https://github.com/Snowflake-Labs/snowflake-vcrpy.git@v0.1.1"
]
```

Reply:
@sfc-gh-klin -- I polled some of our Data Superheros and most said they've never heard of pyproject.toml, after our talk with Berry it seems like it's mainly for people writing client libs. Most of our users are writing apps/jobs so I think the pyproject is unnecessary. We can change direction if we get different feedback from the VS Code PrPr

Reply:
PEP518 was published in 2020 so it makes sense since the recommendation is rather new compared to other packaging tooling. pyproject.toml is intended to be tool and language agnostic given the toml format vs the setup.py python-specific and prescriptive approach. 

I think either way is fine at this stage, yeah.